js_modules/dagster-ui/packages/ui-core/src/runs/filterLogs.tsx

```diff
@@ -49,3 +51,23 @@ export function filterLogs(logs: LogsProviderLogs, filter: LogFilter, filterStep
     textMatchNodes,
   };
 }
+
+// Given an array of metadata entries, returns searchable text in the format:
+// [`label1:value1`, ...], where "value1" is the top-level value(s) present in the
+// metadata entry object converted to JSON. All of our metadata entry types contain
+// different top-level keys, such as "intValue", "mdStr" and "tableSchema", and
+// the searchable text is the value of these keys.
+//
+function metadataEntryKeyValueStrings(node: LogNode) {
+  if (!('metadataEntries' in node)) {
+    return [];
+  }
+  const s = node.metadataEntries.map(
+    // eslint-disable-next-line unused-imports/no-unused-vars
+    ({__typename, label, description, ...rest}) =>
```

Comment:
This might work to sidestep the lint override:

```
({__typename, label: _label, description: _description, ...rest}) =>
```

Reply:
Oh wow didn't know this worked, will change!